src/main/java/uk/gov/companieshouse/database/entity/AddressEntity.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.companieshouse.database.entity;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.springframework.data.annotation.AccessType;
+
+
+import java.io.Serializable;
+
+
```

Comment:
Please remove blank lines:
```suggestion

import java.io.Serializable;

```